assets/js/vue/FileManager.vue

```diff
@@ -83,10 +114,21 @@ let fileManagerResolve = [];
 let totalFiles = 0;
 let doneFiles = 0;
 
+let totalFileSize = 0;
+let doneFileSize = 0;
+
 let myFileManager;
-let myLoadPanel;
 let myDropzone;
 
+let mags = ' KMGTPEZY';
+
+function humanSize(bytes, precision) {
```

Comment:
We could also use our existing function in TwigExtension::formatBytes()

Reply:
hmmm, would be consistent...